docs/processing_information.md

```diff
@@ -64,19 +64,33 @@ Prior to normalization, low-quality cells are removed from the gene by cell coun
 To identify low-quality cells, we use [`miQC`](https://bioconductor.org/packages/release/bioc/html/miQC.html), a package that jointly models proportion of reads belonging to mitochondrial genes and number of unique genes detected.
 Cells with a high likelihood of being compromised (greater than 0.75) and cells that do not pass a minimum number of unique genes detected threshold of 200 are removed from the counts matrix present in the `_processed.rds` file.
 
+If CITE-seq data is present, additional low-quality cells are further removed based on antibody-derived tag (ADT) content, as described in [Processed CITE-seq data](#processed-cite-seq-data).
+
 Log-normalized counts are calculated using the deconvolution method presented in [Lun, Bach, and Marioni (2016)](https://doi.org/10.1186/s13059-016-0947-7).
 The log-normalized counts are used to model variance of each gene prior to selecting the top 2000 highly variable genes (HVGs).
 These HVGs are then used as input to principal component analysis, and the top 50 principal components are selected.
 Finally, these principal components are used to calculate the [UMAP (Uniform Manifold Approximation and Projection)](http://bioconductor.org/books/3.13/OSCA.basic/dimensionality-reduction.html#uniform-manifold-approximation-and-projection) embeddings.
 
+### Processed CITE-seq data
+
+If the experiment contains CITE-seq data, further filtering steps are applied to the `SingleCellExperiment` object found in `_processed.rds`.
+An ambient profile representing antibody-derived tag (ADT) proportions present in the ambient solution is calculated from the `_unfiltered.rds` object using [`DropletUtils::ambientProfileEmpty()`](https://rdrr.io/github/MarioniLab/DropletUtils/man/ambientProfileEmpty.html).
+This ambient profile is then as input to calculate quality-control statistics using [`DropletUtils::cleanTagCounts()`](https://rdrr.io/github/MarioniLab/DropletUtils/man/cleanTagCounts.html).
+If negative control ADTs are present, this information is also provided to `DropletUtils::cleanTagCounts()`.
+`DropletUtils::cleanTagCounts()` identifies cells that should be discarded as those with overly high levels of ambient contamination and/or total negative control counts.
+When cells were filtered based on ADT content, the RNA count matrix was also filtered to match such that the same cells are present in both experiments.
```

Comment:
Shortening this up a bit. I hope that the fact that we are removing data for _cells_ implies that we are removing all data. But I could be convinced that we do need a second sentence clarifying this, but I would avoid the "both experiments" phrasing, as I think that could be confusing as it refers to the data structure, not real "experiments"

```suggestion
Cells identified by `DropletUtils::cleanTagCounts()` as having high levels of ambient contamination and/or negative control tags are are removed from the counts matrices present in the `_processed.rds` file.
```

Reply:
I accepted this and am changing slightly from "from the counts matrices" --> "from all the counts matrices" which I think helps get the point across